README.md

```diff
@@ -128,11 +128,11 @@ Opcodes:
 
 - arithmetic: add, subtract, multiply, (integer) divide, exponent, mod (mathematical)
 - integer comparison: less, greater, etc.
-- indexing: `array_get`, `map_get`, `str_get_byte`
+- indexing: `array_get`, `table_get`, `str_get_byte`
 - conversions: `int_to_string`, `str_to_int`
 - string ops: string concatenation, string split, print, length
-- array/map `length`
-- `map_keys`
+- array/list/table `length`
+- `taable_keys`
```

Comment:
Typo here, but I think a few of these bullets should just be removed because it's already out of sync with code. The README should give a quick overview, then link to a complete list of the builtins, which should just be the code (the Builtin type union) for now until it stabilizes.

Reply:
I simplified the bullets a bit and linked the code.